backend/lib/endpoints/schema/organizations.js

```diff
@@ -28,12 +28,13 @@ const createOrganizationSchema = {
     .prop("global", organization.global.required())
     .prop("industry", organization.industry.required())
     .prop("language", organization.language)
-    .prop("location", organization.location.required())
+    .prop("location", organization.location)
```

Comment:
hmm so isn't this required anymore? I might be missing something, thought it would also have to be required in some validation schema's (e.g. create org)

Reply:
The problem was `organization.location.required()` makes locationSchema.zip required not the whole object. Adding it to required property (see line 37) makes this work as expected while the other primitive required properties also still work.